src/Workspaces/CSharp/Portable/Simplification/CSharpSimplificationService.cs

```diff
@@ -90,6 +91,11 @@ public static SyntaxToken TryEscapeIdentifierToken(SyntaxToken syntaxToken, Synt
                 return syntaxToken;
             }
 
+            if (SyntaxFacts.GetContextualKeywordKind(syntaxToken.ValueText) == SyntaxKind.UnderscoreToken)
+            {
+                return syntaxToken;
+            }
```

Comment:
Is the right place for this to happen? If the problem is that the expander is incorrectly escaping discards (i.e. `_` -> `@_`), shouldn't the fix go in the expander?